module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -1282,7 +1282,10 @@ public function getFormatClassifications()
      */
     public function getIdentifier()
     {
-        return $this->fields['identifier'] ?? [];
+        foreach ($this->getIdentifiersByType(true) as $identifier) {
+            return [$identifier];
```

Comment:
Tämän varmaan pitäisi palauttaa kaikki identifier:t eli suoraan se, mitä `getIdentifiersByType` palauttaa. `identifier`-kenttä on indeksissä multiValued, eli niitä voi olla monta. Metodi vaan on nimetty hölmösti..

Reply:
Hmm... Kun LIDOlla on ollut niin että Tunniste-kohdassa on näkynyt vain eka tunniste, ja Muu tunniste -kohdassa muut. Aiemmin toimi näin siksi, että identifieriin on indeksoitu vain eka. Eli jos muutetaan niin että tähän laitetaankin kaikki, niin sitten se Muu tunniste kohta on toistoa.

Reply:
Ah, okei. No sitten kommentti tohon, miksi otetaan vain eka.

Reply:
Ok, lisäsin